src/scenes/home/faq/question/question.js

```diff
@@ -8,6 +8,7 @@ class Question extends Component {
       <div className={styles.accordionSingle}>
         <input className={styles.accordionSingleHidden} type="checkbox" id={this.props.q} />
         <label className={styles.accordionSingleQuestion} htmlFor={this.props.q}> {this.props.q}</label>
+        { // eslint-disable-next-line react/no-danger react/prop-types: 0) }
```

Comment:
This is what ended up working:
```
      <label className={styles.accordionSingleQuestion} htmlFor={this.props.q}> {this.props.q}</label>
        {   // eslint-disable-next-line react/no-danger react/prop-types: 0) }
            // eslint-disable-next-line
          <p className={styles.accordionSingleAnswer} dangerouslySetInnerHTML={{ __html: this.props.a }} />
      }</div>
```
The closing tag was in a comment, so I just had to put the closing bracket on another line.